lib/registration.pm

```diff
@@ -733,6 +733,9 @@ sub select_addons_in_textmode {
 sub registration_bootloader_cmdline {
     # https://www.suse.com/documentation/smt11/book_yep/data/smt_client_parameters.html
     # SCC_URL=https://smt.example.com
+    # prevent rogue RMT servers to show up in unexpected selection dialogs
+    # https://progress.opensuse.org/issues/94696
+    set_var('SCC_URL', 'https://scc.suse.com') unless get_var('SCC_URL');
```

Comment:
this method seems that doesn't run for all bootloaders, wdyt about https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/products/sle/main.pm#L360 ?

Reply:
well, https://openqa.suse.de/tests/6329021 looks fine so I guess we are ok. If someone duplicates the regurl typing on the command line then we should find these and apply proper fixes. But I doubt we will be able to find such cases before we should merge this PR

Reply:
will go through this function when using 'bootloader_start' test module? I'm checking but no so sure...

Reply:
At this point I don't care as long as we fix most of the other cases